### DISCUSSION TITLE
Doris official website adds a new release page

### DISCUSSION THREAD

**Discussion by hf200012** [apache/doris#7016] [Ideas]
We are going to add a list of the latest developments on the doris official website. Doris’s release news, best practices, use cases, and related technical articles can all be quickly published in the form of Markdown, separated from the current Doris document for easy reference. 

The following is the design plan and ideas, I hope to get your feedback and opinions，Attached is a sample of an article.

The design plan is to add an article directory under the zh_cn, en directory of the current Docs directory. You only need to edit the article in markdown mode and put it in the changed directory. All Markdown files come out with Apache License header, and you need to carry them. The following piece of information:
```
{
     "title": "article title",
     "description": "article description",
     "date": "2021-11-03",
     "metaTitle": "article",
     "language": "zh-CN",
     "layout": "Article",
     "sidebar": false
}
```

Similar effects are as follows:
![img](https://raw.githubusercontent.com/hf200012/images/master/1.png)
Click the latest news：（Support paging）
![img](https://raw.githubusercontent.com/hf200012/images/master/3.png)
View article content：
![img](https://raw.githubusercontent.com/hf200012/images/master/2.png)

**Comment by morningman** [comment]
Good idea
